set.mm

```diff
@@ -667621,6 +668593,52 @@ proof usage (see ~ mplelbas and ~ mpladd ).  Note that hypothesis 0 is
       UUQUDUWGUJYOAEHJKMNOPWKQRUBUDUGUIUJUKULYPYQYR $.
   $}
 
+  ${
+    $d A b i $.  $d D b i $.  $d D g $.  $d F b $.  $d G b $.  $d I b h $.
+    $d I i $.  $d K b i $.  $d N g $.  $d R b $.  $d S b i $.  $d U b h $.
+    $d U i $.  $d ph b i $.  $d g h $.
+    evlsmhpvvval.q $e |- Q = ( ( I evalSub S ) ` R ) $.
+    evlsmhpvvval.p $e |- H = ( I mHomP U ) $.
+    evlsmhpvvval.u $e |- U = ( S |`s R ) $.
+    evlsmhpvvval.d $e |- D = { h e. ( NN0 ^m I ) | ( `' h " NN ) e. Fin } $.
+    evlsmhpvvval.g $e |- G = { g e. D | ( ( CCfld |`s NN0 ) gsum g ) = N } $.
+    evlsmhpvvval.k $e |- K = ( Base ` S ) $.
+    evlsmhpvvval.m $e |- M = ( mulGrp ` S ) $.
+    evlsmhpvvval.w $e |- .^ = ( .g ` M ) $.
+    evlsmhpvvval.x $e |- .x. = ( .r ` S ) $.
+    evlsmhpvvval.i $e |- ( ph -> I e. V ) $.
+    evlsmhpvvval.s $e |- ( ph -> S e. CRing ) $.
+    evlsmhpvvval.r $e |- ( ph -> R e. ( SubRing ` S ) ) $.
+    evlsmhpvvval.n $e |- ( ph -> N e. NN0 ) $.
+    evlsmhpvvval.f $e |- ( ph -> F e. ( H ` N ) ) $.
+    evlsmhpvvval.a $e |- ( ph -> A e. ( K ^m I ) ) $.
+    $( Give a formula for the evaluation of a homogeneous polynomial given
+       assignments from variables to values.  The difference between this and
+       ~ evlsvvval is that ` b e. D ` is restricted to ` b e. G ` .
```

Comment:
...and putting `b e. D` instead of `b e. G` means that the sum of all variable degrees is `N` everywhere, so that the polynomial is homogeneous.